qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -755,3 +757,111 @@ def test_refuse_client_session_on_reconnect(self):
             self.assertEqual(len(self.fs.mds_tell(["session", "ls"])), 0)
         self.mount_a.umount_wait(force=True)
 
+
+class TestClientOnLaggyOSD(CephFSTestCase):
+    CLIENTS_REQUIRED = 2
+
+    LOAD_SETTINGS = ["mds_reconnect_timeout", "ms_max_backoff"]
+
+    # Environment references
+    mds_reconnect_timeout = None
+    ms_max_backoff = None
+
+    TEST_OSD = 0
```

Comment:
The same OSD marked laggy needs to be used in helper `clear_laggy_params` to make it unlaggy. Also 0 is the sure shot number in case even if we have a cluster with 1 OSD

Reply:
I understand that. If you chose a random OSD in the test, that OSD id is valid for the entire test. Wouldn't that suffice?

Reply:
There's a list `live_osds` in ceph_manager.py, I'll use it with random.choice to fetch an random live in both testcases

Reply:
done, PTAL